src/env.py

```diff
@@ -9,7 +9,7 @@
 from shutil import rmtree, copytree, copyfile
 from subprocess import PIPE, Popen, CalledProcessError
 from traceback import format_exc
-
+from random import randint
```

Comment:
As this commit add functionality to add new user rather than updating existing user on the server commit message should be improved. "Add new krb user if username  already exists on ipa server".